src/components/themed/SideMenu.tsx

```diff
@@ -163,18 +163,18 @@ export function SideMenuComponent(props: Props) {
   }
 
   const handleShareApp = () => {
-    const message = `${sprintf(lstrings.share_subject, config.appName)}\n\n${lstrings.share_message}\n\n`
-
     // Generate anonymized referral ID
     const data = Uint8Array.from(Buffer.from(account.rootLoginId, 'hex'))
     const refId = hashjs.sha256().update(data).digest('hex').replace('0x', '').substring(0, 10)
 
-    const website = `${config.website}?af=appreferred_${refId}`
+    const url = `${config.website}?af=appreferred_${refId}`
 
     const shareOptions = {
       failOnCancel: false,
-      message: Platform.OS === 'ios' ? message : message + website,
-      url: Platform.OS === 'ios' ? website : ''
+      title: lstrings.share_subject,
```

Comment:
The `share_subject` string has a `%s` in it, so you need to bake it out. While we're here, let's also use async / await to get the debounce effect:

```ts
const subject = sprintf(lstrings.share_subject, config.appName)
await Share.open({
  subject,
  title: subject,
  ...
})
```